### PR TITLE
Add an error for >1 dtar rank

### PR DESCRIPTION
**Pull request by gonsie**

### CODE REVIEW

**Comment by gonsie** [comment]
Shoot, I did this on my phone and forgot the sign off. I’ll force add that next time I’m at a computer.

**Comment by dsikich** [comment]
@gonsie your rules ;)

File: src/dtar/dtar.c [hunk 1]
```diff
@@ -95,6 +95,15 @@ int main(int argc, char** argv)
     MPI_Comm_rank(MPI_COMM_WORLD, &rank);
     MPI_Comm_size(MPI_COMM_WORLD, &size);
 
+    if (size > 1) {
+        if (rank == 0) {
+            printf(“ERROR: dtar only works with one MPI rank.\n”);
```
Comment (adammoody) [inline src/dtar/dtar.c]: `MFU_LOG(MFU_LOG_ERR, "dtar currently only works with one MPI rank");`

Comment (adammoody) [reply src/dtar/dtar.c]: Figured we can use our MFU_LOG function here.

**Comment by gonsie** [comment]
thanks @adammoody should be good (if it builds)

**Pull request merged**